gstools/variogram/estimator.pyx

```diff
@@ -345,8 +332,8 @@ def ma_structured(
     cdef int j_max = f.shape[1]
     cdef int k_max = i_max + 1
 
-    cdef vector[double] variogram = vector[double](k_max, 0.0)
-    cdef vector[long] counts = vector[long](k_max, 0)
+    cdef double[:] variogram = np.zeros(k_max)
+    cdef long[:] counts = np.zeros(k_max, dtype=long)
```

Comment:
Ah! So you need to specify dtype if it's not `double` right? I hope numpy doesn't change its default behavior to soon.